packages/react-components/react-tree/stories/Tree/FlatTree.stories.tsx

```diff
@@ -2,131 +2,45 @@ import * as React from 'react';
-
-      <MenuPopover>
-        <MenuList>
-          <MenuItem>New</MenuItem>
-          <MenuItem>New Window</MenuItem>
-          <MenuItem disabled>Open File</MenuItem>
-          <MenuItem>Open Folder</MenuItem>
-        </MenuList>
-      </MenuPopover>
-    </Menu>
-  </>
-);
 
 export const FlatTreeStory = () => {
-  const flatTree = useHeadlessFlatTree_unstable(flatTreeItems);
-  const focusTargetAttribute = useRestoreFocusTarget();
-
+  const [openItems, setOpenItems] = React.useState<Set<TreeItemValue>>(() => new Set());
+  const handleOpenChange = (event: TreeOpenChangeEvent, data: TreeOpenChangeData) => {
+    setOpenItems(data.openItems);
+  };
   return (
-    <FlatTree {...flatTree.getTreeProps()} aria-label="Flat Tree">
-      {Array.from(flatTree.items(), flatTreeItem => {
-        const { content, ...treeItemProps } = flatTreeItem.getTreeItemProps();
-        return (
-          <Menu key={flatTreeItem.value} positioning="below-end" openOnContext>
-            <MenuTrigger disableButtonEnhancement>
-              <FlatTreeItem aria-description="has actions" {...focusTargetAttribute} {...treeItemProps}>
-                <TreeItemLayout actions={<ActionsExample />}>{content}</TreeItemLayout>
-              </FlatTreeItem>
-            </MenuTrigger>
-            <MenuPopover>
-              <MenuList>
-                <MenuItem>Edit</MenuItem>
-                <MenuItem>New</MenuItem>
-                <MenuItem>New Window</MenuItem>
-                <MenuItem disabled>Open File</MenuItem>
-                <MenuItem>Open Folder</MenuItem>
-              </MenuList>
-            </MenuPopover>
-          </Menu>
-        );
-      })}
+    <FlatTree openItems={openItems} onOpenChange={handleOpenChange} aria-label="Flat Tree">
+      <FlatTreeItem value="1" aria-level={1} aria-setsize={2} aria-posinset={1} itemType="branch">
+        <TreeItemLayout>Item 1, level 1</TreeItemLayout>
+      </FlatTreeItem>
+      {openItems.has('1') && (
+        <>
+          <FlatTreeItem parentValue="1" value="1-1" aria-level={2} aria-setsize={2} aria-posinset={1} itemType="leaf">
+            <TreeItemLayout>Item 1, level 2</TreeItemLayout>
+          </FlatTreeItem>
+          <FlatTreeItem parentValue="1" value="1-2" aria-level={2} aria-setsize={2} aria-posinset={2} itemType="leaf">
+            <TreeItemLayout>Item 1, level 2</TreeItemLayout>
+          </FlatTreeItem>
+        </>
+      )}
+      <FlatTreeItem value="2" aria-level={1} aria-setsize={2} aria-posinset={1} itemType="branch">
+        <TreeItemLayout>Item 1, level 1</TreeItemLayout>
+      </FlatTreeItem>
+      {openItems.has('2') && (
+        <>
+          <FlatTreeItem parentValue="2" value="2-1" aria-level={2} aria-setsize={2} aria-posinset={1} itemType="leaf">
+            <TreeItemLayout>Item 1, level 2</TreeItemLayout>
+          </FlatTreeItem>
+          <FlatTreeItem parentValue="2" value="2-2" aria-level={2} aria-setsize={2} aria-posinset={2} itemType="leaf">
```

Comment:
nit: would be nice to have three items there to clearly show that `aria-set-size` is not something static